src/modules/hyprland/workspaces.cpp

```diff
@@ -861,6 +872,7 @@ void Workspace::update(const std::string &format, const std::string &icon) {
   addOrRemoveClass(styleContext, isPersistent(), "persistent");
   addOrRemoveClass(styleContext, isUrgent(), "urgent");
   addOrRemoveClass(styleContext, isVisible(), "visible");
+  addOrRemoveClass(styleContext, m_workspaceManager.getBarOutput() == output(), "onThisMonitor");
```

Comment:
I don't like `active-monitor` since it also gets applied to inactive workspaces and inactive monitors.

Maybe `correct-monitor`. correct meaning that `waybar output monitor == workspace output monitor`.
That's still not very self-explanatory though. What do you think?

Reply:
what about `is-same-monitor`? :D 
Otherwise go with `correct-monitor` and properly document it in man and wiki :)

Reply:
Maybe `hosting-monitor` or `parent-monitor`. I quite like `hosting-monitor`.